dev/packages/alpha/mysql/0.0.2/dataset/error/manifest.yml

```diff
@@ -1,5 +1,4 @@
 title: MySQL error logs
-release: ga
```

Comment:
I think our default is experimental. So if you remove it completely, it will be experimental by default.

Reply:
Hm, thought about this again. We don't have this feature yet on the Kibana side so it will show them.

Reply:
Ah, right. Fixed.

Reply:
Nevertheless I will leave them marked as "beta".